ykbuild/build.rs

```diff
@@ -0,0 +1,52 @@
+use cmake;
+use std::env;
+use num_cpus;
+use which::which;
+
+const YKLLVM: &str = "../ykllvm/llvm";
+const PROFILE: &str = "Release";
+
+fn main() {
+
+    if env::var("YKLLVM_INSTALL_DIR").is_ok() {
+        println!("cargo:ykllvm={}", env::var("YKLLVM_INSTALL_DIR").unwrap());
+        return;
+    }
+
+    // Ninja builds are faster, so use this if it's available.
+    let use_ninja = which("ninja").is_ok();
+    let is_debug = env::var("PROFILE").unwrap() == "debug";
+    let nprocs = format!("-j {}", num_cpus::get());
+
+    let mut ykllvm = cmake::Config::new(YKLLVM);
+    ykllvm.profile(PROFILE)
+        .generator(if use_ninja { "Ninja" } else { "Unix Makefiles" })
+        .define("LLVM_INSTALL_UTILS", "ON")
+        .define("LLVM_ENABLE_PROJECTS", "lld;clang")
+        .define("LLVM_ENABLE_ASSERTIONS", if is_debug { "ON" } else { "OFF" })
+        // Due to an LLVM bug, PIE breaks our mapper, and it's not enough to pass
+        // `-fno-pie` to clang for some reason:
+        // https://github.com/llvm/llvm-project/issues/57085
+        .define("CLANG_DEFAULT_PIE_ON_LINUX", "OFF")
+        .build_arg(nprocs);
+
+    match env::var("YKLLVM_BUILD_ARGS") {
```

Comment:
I think this will be easier as `if let Ok(args) = env::var("...")`?

Reply:
Fixed here 4371375ffbc3d273b344e1df8f9f9bc9c85f2c24